pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1262,6 +1262,10 @@ func (l *LibvirtDomainManager) syncNetworkHotplug(
 	if err := networkInterfaceManager.hotUnplugVirtioInterface(vmi, &api.Domain{Spec: *oldSpec}); err != nil {
 		return err
 	}
+	if err := networkInterfaceManager.updateDomainLinkState(&api.Domain{Spec: *oldSpec}, domain); err != nil {
```

Comment:
nit: possible follow-up:
Consider using the `DomainSpec` as a parameter instead of `Domain`.
Same for `hotplugVirtioInterface`